flybywiresim/a32nx
# Tail Camera

Fixes #[issue_no]

## Summary of Changes
Hi can you add this camera view in lower ECAM on A320?
## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
![125379910_a340tc jpg 35f5783840a6b945062b3ff801787c14](https://user-images.githubusercontent.com/74202232/99011782-86dcd780-2555-11eb-83d8-4c8165e3df6d.jpg)
###
